core/src/test-core/java/com/radixdlt/rev2/REv2TestTransactions.java

```diff
@@ -131,7 +131,7 @@ public static String constructNewAccountManifest(NetworkDefinition networkDefini
                     CALL_METHOD ComponentAddress("%s") "lock_fee" Decimal("100");
                     CALL_METHOD ComponentAddress("%s") "free";
                     TAKE_FROM_WORKTOP ResourceAddress("%s") Bucket("xrd");
-                    CALL_FUNCTION PackageAddress("%s") "Account" "new_with_resource" Enum("AllowAll") Bucket("xrd");
+                    CALL_FUNCTION PackageAddress("%s") "Account" "new_with_resource" Enum(0u8) Bucket("xrd");
```

Comment:
For clarity, I think you can do `Enum("AccessRule::AllowAll")` here, and on line 155

Reply:
ah, did not know this was possible!